### DISCUSSION TITLE
Error while running as docker

### DISCUSSION THREAD

**Discussion by DrBartschi** [remsky/Kokoro-FastAPI#219] [General]
I tried to run it as a docker but theres some errors:

Error generating speech: MediaSource.addSourceBuffer: Type not supported in MediaSource

Log:
/app/.venv/lib/python3.10/site-packages/torch/nn/modules/rnn.py:123: UserWarning: dropout option adds dropout after all but last recurrent layer, so non-zero dropout expects num_layers greater than 1, but got dropout=0.2 and num_layers=1

  warnings.warn(

/app/.venv/lib/python3.10/site-packages/torch/nn/utils/weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.

  WeightNorm.apply(module, name, dim)

**Comment by mynona** [comment]
same

**Reply by DrBartschi** [reply]
I had the error with the Firefox browser. If I use Google Chrome, for example, it works now
